### PR TITLE
Just version

### PR DESCRIPTION
**Pull request by Janamou**

### CODE REVIEW

**Comment by kevmoo** [comment]
Do tests pass on your machine?

**Comment by Janamou** [comment]
@kevmoo ok, interesting. If I run all the tests with `pub run test` then the Firestore starts to have timeouts problems even locally. If I run it as `pub run test test/firestore_test.dart` (or any file), all tests are passed. Can pub run all the tests but not in parallel?

**Comment by Janamou** [comment]
@janamou @kevmoo yes, it can, `pub run test --concurrency=1`. Going to try on Travis.

**Comment by Janamou** [comment]
@kevmoo aaaah, fixed with the concurrency 😂 🎉  https://travis-ci.org/Janamou/firebase-dart

File: .travis.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ addons:
 before_script: ./tool/travis_before_script.sh
 dart_task:
   - test: --platform vm
-  - test: --platform firefox
-  - test: --platform dartium
+  - test: --platform firefox --concurrency=1
```
Comment (kevmoo) [inline .travis.yml]: Ideally we wouldn't share names between each test – this helps, too

**Pull request merged**